### DISCUSSION TITLE
NPM error when installing any package

### DISCUSSION THREAD

**Discussion by zacjones93** [johnlindquist/kit#754] [Errors]
```bash
Error: /Users/zacjones/.kenv/node_modules/text-case/package.json doesn't exist
Error: Error: /Users/zacjones/.kenv/node_modules/text-case/package.json doesn't exist
    at kenvImport (file:///Users/zacjones/.kit/api/npm.js:44:15)
    at async file:///Users/zacjones/.kit/api/npm.js:60:12
    at async file:///Users/zacjones/.kenv/scripts/switch-case.js
```

For some reason, whenever I try to install a dependency in a script I get this error that package.json doesn't exist. I am on a brand new machine and might not have my system properly set up.

If I manually npm install in the `.kenv` directory everything works great.

Script below:
```js
// Name: switch-case
// Shortcut: shift control C

import "@johnlindquist/kit"
let { titleCase, sentenceCase, paramCase, camelCase } = await npm("text-case");

let text = await getSelectedText();

let options = {
  "Title Case": titleCase,
  "Sentence case": sentenceCase,
  "param-case": paramCase,
  camelCase: camelCase,
};

let option = await arg(
  "What case do you want to convert to?",
  Object.keys(options)
);

let formattedText = options[option](text);

await setSelectedText(formattedText);

```

**Comment by johnlindquist** [comment]
Do you see these two screens like it's attempting to install? Or does it just error?

<img width="808" alt="CleanShot 2022-03-30 at 13 45 45@2x" src="https://user-images.githubusercontent.com/36073/160918279-6a6f6158-2db1-4432-bd79-0a811aaa0b8e.png">
<img width="812" alt="CleanShot 2022-03-30 at 13 45 55@2x" src="https://user-images.githubusercontent.com/36073/160918295-cef27610-4605-443c-97cc-e62682a84360.png">

**Reply by zacjones93** [reply]
I see the first but not the second screen

![Image 2022-03-31 at 10 23 05 AM](https://user-images.githubusercontent.com/6188161/161078397-8f84dd55-aff8-429a-89df-9a0ab909df14.jpg)
![Image 2022-03-31 at 10 23 24 AM](https://user-images.githubusercontent.com/6188161/161078419-dc820aed-696b-43e0-aeb6-ab69984d165a.jpg)

**Comment by cfritze64** [comment]
I'm seeing the same error as Zac - also first screen as expected, but then package.json doesn't exist error

**Comment by cfritze64** [comment]
resolved. I broke a dependency for python. Lots of help from John in the background. thx
